web/controllers/courseController/index.ts

```diff
@@ -2805,23 +2808,67 @@ async function groupMail(student: Student, courseId: number, subcourseId: number
         return 403;
     }
 
+    const lectures = subcourse.lectures.sort(
+        (a, b) => a.start.getMilliseconds() - b.start.getMilliseconds()
+    );
+    const lastLecture = lectures[lectures.length - 1];
+    if (lastLecture != null) {
```

Comment:
I would either prefer strict equality comparison (`!==`) to `undefined` here, or alternatively just `if (lastLecture)` ... Comparing to `null` makes it seem as if the value could possibly be null ...